aptos-move/block-executor/src/liftings.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+//! Implementation of sync and unsync versions of data structures which are used
+//! to process aggregator liftings.
+//!
+//! Both have:
+//!   1. A counter used to generate a new unique (per-block) identifier.
+//!   2. A map from identifiers to lifted Move values.
+
+use dashmap::{mapref::one::Ref, DashMap};
+use move_vm_types::values::Value;
+use std::{
+    collections::HashMap,
+    sync::atomic::{AtomicU64, Ordering},
+};
+
+pub(crate) struct UnsyncLiftings {
+    counter: u64,
+    lifted_values: HashMap<u64, Value>,
```

Comment:
Just wondering about the terminology. Why are they called lifted values?

Reply:
We called them AggregatorLiftings before, so I used that. Didn't use the first aggregator part, should I?